dev.goats.xivlauncher.yml

```diff
@@ -58,20 +69,23 @@ modules:
       - install -D -m644 misc/linux_distrib/512.png "${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/xivlauncher.png"
       - install -D -m644 dev.goats.xivlauncher.appdata.xml "${FLATPAK_DEST}/share/metainfo/dev.goats.xivlauncher.appdata.xml"
       - ln -s "${FLATPAK_DEST}/opt/XIVLauncher/XIVLauncher.Core" "${FLATPAK_DEST}/bin/xivlauncher"
+
     sources:
-      - type: git
-        url: https://github.com/goatcorp/XIVLauncher.Core.git
-        commit: 599241d8febafaa46c7111362560a6af361eca48
-        tag: "1.0.2"
+      - type: archive
+        url: https://github.com/goatcorp/XIVLauncher.Core/archive/refs/tags/1.0.2.tar.gz
+        sha256: ec27c7c6b018d5d8f25a47a946b0fe811fb84f5ec37af3c07884e1adccf22f73
+
```

Comment:
This part shouldn't be changed. The archive does not contain the submodule, so you'd have to download that separately, put it in the correct place, and then you'd have to modify the `dotnet publish` command to pass it a build hash, otherwise it will fail since it's expecting a git repo.